### ISSUE TITLE
[QUESTION] theme for each user

### ISSUE THREAD

**Issue by irasdesire** [Lissy93/dashy#1574]
### Question

Hello! i have a question about dashy theme;

context: i m using dashy with keycloak so i didn t have a user type, i changed somehow the isUserAdmin() function to return isAdmin based on usermane 🤐.  and now the admin users have the theme i chose in the config.yml and the rest of users have the default theme.

question: can anyone tell me where i can see if each user has a theme set somewhere so i can go and change there so every user could have the theme i choose for them?

i can provide more info if needed, hope someone would help me and thank you in advance! 🥳

### Category

Setup and Deployment

### Please tick the boxes

- [X] You are using a [supported](https://github.com/Lissy93/dashy/blob/master/.github/SECURITY.md#supported-versions) version of Dashy (check the first two digits of the version number)
- [X] You've checked that this [question hasn't already been raised](https://github.com/Lissy93/dashy/issues?q=is%3Aissue)
- [X] You've checked the [docs](https://github.com/Lissy93/dashy/tree/master/docs#readme) and [troubleshooting](https://github.com/Lissy93/dashy/blob/master/docs/troubleshooting.md#troubleshooting) guide
- [X] You agree to the [code of conduct](https://github.com/Lissy93/dashy/blob/master/.github/CODE_OF_CONDUCT.md#contributor-covenant-code-of-conduct)

**Comment by Lissy93** [comment]
Hey @irasdesire 
You can have a theme for each page, and a locally stored theme on each browser (so different users could have different themes that way, if they have different computers), but right now there's no option for a theme for each user.

Realistically, I don't think this will be a feature that I implement, as the complexity would be quite high, for something quite niche.

**Comment by irasdesire** [comment]
hey, thank you for the answear! maybe i wasn t clear in my question, the problem is that i want only one theme for everyone and for some reason admin user see a theme, the rest of the users see other theme (the default one) and i don t know how comes this behavior.
the change i made : for the config, in SettingsContainer.vue i added a v-if based on computed isUserAdmin() . so, only if the user is admin, he can see    <ThemeSelector />, <LayoutSelector  />, <ItemSizeSelector  />, <ConfigLauncher />. for the admin user everything is fine, for the rest of the users it shows the default theme....also i added some console.logs and i see that the theme looks corectly set (i chose material) still the default one is shown in browser :( maybe i did something wrong i don t know, maybe you can make it clear for me

**Comment by CrazyWolf13** [comment]
So you are speaking of a config option to set a default theme globally, for all users?

**Comment by irasdesire** [comment]
yes, i set it using theme selector (chose material) and other users still se the default one + also in config.yml i made theme: material

**Comment by Lissy93** [comment]
Ah, I see. Yes, if you set a theme in your config, that will be applied for everyone / all users.

When a user changes the theme through the UI, their chosen theme is remembered in their browsers local storage.
So if you're seeing a different theme, to what you set in the config, try clearing local storage (under the Config Menu --> Reset Local Settings)

**Comment by lissy93** [comment]
Closing, as I think this questions is answered. Let me know if it's not, and I can re-open.
